ingest/ingest_files.py

```diff
@@ -298,6 +311,15 @@ def open_pandas(self, file_path, file_type, **kwargs):
         else:
             raise ValueError("File must be tab or comma delimited")
 
+    def open_h5ad(self, file_path, **kwargs):
+        """Opens file as AnnData object """
+        try:
+            return sc.read_h5ad(file_path, backed='r')
+        except OSError as e:
+            msg = f"Unable to read file, \"{file_path}\", using scanpy."
```

Comment:
Formatting errors as "{general message} {specific thing}" helps parsing, both visually and programmatically.  The particular wording suggested below is less relevant than ensuring the specific is completely after the general.

```suggestion
            msg = f"Scanpy cannot read file \"{file_path}\""
```

Reply:
`d52dd5b`